src/noit_mtev_bridge.h

```diff
@@ -33,9 +33,13 @@
 
 #include <mtev_log.h>
 
-#define noit_debug mtev_debug
-#define noit_error mtev_error
-#define noit_notice mtev_notice
+mtev_log_stream_t noit_error_impl;
+mtev_log_stream_t noit_notice_impl;
+mtev_log_stream_t noit_debug_impl;
```

Comment:
These must be `extern` in a header and the actual declarations in a C file somewhere.  This is where it gets more complicated b/c of libnoit.

Reply:
Updated. Now running into this error:

```
noit_check_log_helpers.o: In function `noit_check_log_bundle_compress_b64':
//home/vagrant/circ/src/reconnoiter/src/noit_check_log_helpers.c:101: undefined reference to `noit_error_impl'
//home/vagrant/circ/src/reconnoiter/src/noit_check_log_helpers.c:101: undefined reference to `noit_error_impl'
//home/vagrant/circ/src/reconnoiter/src/noit_check_log_helpers.c:101: undefined reference to `noit_error_impl'
//home/vagrant/circ/src/reconnoiter/src/noit_check_log_helpers.c:101: undefined reference to `noit_error_impl'
//home/vagrant/circ/src/reconnoiter/src/noit_check_log_helpers.c:101: undefined reference to `noit_error_impl'
noit_check_log_helpers.o://home/vagrant/circ/src/reconnoiter/src/noit_check_log_helpers.c:101: more undefined references to `noit_error_impl' follow
```

Reply:
noit_check_log_helpers.c does include noit_mtev_bridge.h, but apparently the linker can not resolve the noit_error_impl symbol. Do we need to add a linker flag here `-Lsomething` ?

Reply:
Tweaked the Makefile a little bit. Seems to build now on my box.

Reply:
The tricky part here is that those symbols need to be in one object and not in more than one.  We have the library and the binaries (noitd, stratcond, noit_b2sm).  This will require a trick to pull off.

there's also the question in that if I use libnoit (from say snowthd), should it be logging to these "noit/debug" or should it be to general debug?

Reply:
I certainly want libnoit to log into substreams `error/noit`, etc., so that I can keep an eye on `error` and selectively disable noit logs when I am not interested in them.

Have you seen the changes to the makefile? Looks like this is working as intended. I had a few double definition errors but got them resolved.